oauth2/oauth2_auth_code_test.go

```diff
@@ -45,6 +46,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"golang.org/x/oauth2"
+	goauth2 "golang.org/x/oauth2"
 	"golang.org/x/oauth2/clientcredentials"
 
 	"github.com/ory/fosite"
```

Comment:
@aeneasr The error is in CI as well, so we probably forgot to run quicktest after removing the `// swagger:ignore` directives

LINE 135 in this file:
```
--- FAIL: TestAuthCodeWithDefaultStrategy (9.01s)
    --- FAIL: TestAuthCodeWithDefaultStrategy/case=respects_client_token_lifespan_configuration (0.09s)
        --- FAIL: TestAuthCodeWithDefaultStrategy/case=respects_client_token_lifespan_configuration/case=custom-lifespans-active-jwt (0.09s)
            oauth2_auth_code_test.go:135:
                        Error Trace:    oauth2_auth_code_test.go:135
                                                                server.go:2047
                                                                server.go:2879
                                                                server.go:1930
                                                                asm_amd64.s:1581
                        Error:          Received unexpected error:
                                        json: cannot unmarshal string into Go struct field OAuth2Client.client.implicit_grant_access_token_lifespan of type models.NullDuration
                        Test:           TestAuthCodeWithDefaultStrategy/case=respects_client_token_lifespan_configuration/case=custom-lifespans-active-jwt
            oauth2_auth_code_test.go:124:
                        Error Trace:    oauth2_auth_code_test.go:124
                                                                oauth2_auth_code_test.go:407
                                                                oauth2_auth_code_test.go:460
                        Error:          Received unexpected error:
                                        Get "http://127.0.0.1:40185/?login_challenge=8339eb8eaae04cf69556fb80e8c7c86d": EOF
                        Test:           TestAuthCodeWithDefaultStrategy/case=respects_client_token_lifespan_configuration/case=custom-lifespans-active-jwt

```